app/App.js

```diff
@@ -1,18 +1,42 @@
 import React from "react"
 import { registerRootComponent } from "expo"
 import { NavigationContainer } from "@react-navigation/native"
+import { View, Text, StyleSheet } from "react-native"
 
+import { useFonts } from "expo-font"
 import Routes from "@navigation/Routes"
+import COLORS from "@utils/colors"
 
 
-class App extends React.Component {
-    render() {
-        return (
+const styles = StyleSheet.create({
+    app: {
+        height: "100%",
+    },
+    wordmark: {
+        textAlign: "center",
+        fontFamily: "CairoBold",
+        fontSize: 16,
+        color: COLORS.green,
+        backgroundColor: COLORS.lightGreen,
+    },
+})
+
+
+const App = () => {
+    const [fontsLoaded] = useFonts({
+        "CairoBold": require("./assets/fonts/Cairo-Bold.ttf"),
+        "CairoRegular": require("./assets/fonts/Cairo-Regular.ttf"),
+    })
```

Comment:
is it possible to create one file with all fonts and then use import where we want to use them ?

Reply:
Are you asking can we put all ttf files in one? - Answer is no.
If you are asking should we load fonts in every component? - Answer is no. You don't need to load fonts in every component. Fonts are loading just once in the main component and now you can use everywhere in the application.